.github/workflows/main.yml

```diff
@@ -72,14 +72,8 @@ jobs:
       - uses: actions/checkout@v4
         with:
           fetch-depth: 0
```

Comment:
Do you just want to move this into the new checkout action?

Reply:
I initially tried that yeah but ended up finding that using `./.github/...` requires checking out first so this was required to be split out (alas)